Changelog.md

```diff
@@ -70,6 +71,8 @@
     * `mapM` to `rmapM`
     * `sequence` to `rsequence`
 
+* Remove `Applicative` instance of folds.
```

Comment:
This should go in breaking changes, not in deprecations.

Reply:
Also, add "Please use `teeWith` or the `Tee` type as an alternative to Fold applicative.".